hack/install-kwok.sh

```diff
@@ -73,6 +73,7 @@ cat <<EOF > "${SUB_LET_DIR}/kustomization.yaml"
   - name: registry.k8s.io/kwok/kwok
     newTag: "${KWOK_LATEST_RELEASE}"
   resources:
+#  - ./metrics-resource.yaml
```

Comment:
Do we need this? Should it be removed or the comment removed?

Reply:
We can add it back later, trying to figure out how to enable the kwok metrics endpoint